watertap/unit_models/zero_order/tests/test_ozone_aop_zo.py

```diff
@@ -0,0 +1,382 @@
+    check_optimal_termination, ConcreteModel, Constraint, value, Var)
+from pyomo.util.check_units import assert_units_consistent
+
+from idaes.core import FlowsheetBlock
+from idaes.core.util.exceptions import ConfigurationError
+from idaes.core.util import get_solver
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core.util.testing import initialization_tester
+
+from watertap.unit_models.zero_order import OzoneAOPZO
+from watertap.core.wt_database import Database
+from watertap.core.zero_order_properties import WaterParameterBlock
+
+solver = get_solver()
+
+
+class TestOzoneZO_with_default_removal:
+    @pytest.fixture(scope="class")
+    def model(self):
+        m = ConcreteModel()
+        m.db = Database()
+
+        m.fs = FlowsheetBlock(default={"dynamic": False})
+        m.fs.params = WaterParameterBlock(default={"solute_list": ["cryptosporidium", 
+                                                                    "toc", 
+                                                                    "giardia_lamblia",
+                                                                    "eeq", 
+                                                                    "total_coliforms_fecal_ecoli", 
+                                                                    "viruses_enteric",
+                                                                    "tss"]})
+
+        m.fs.unit = OzoneAOPZO(default={ "property_package": m.fs.params,
+                                            "database": m.db})
+
+        m.fs.unit.inlet.flow_mass_comp[0, "H2O"].fix(100)
+        m.fs.unit.inlet.flow_mass_comp[0, "cryptosporidium"].fix(1)
+        m.fs.unit.inlet.flow_mass_comp[0, "toc"].fix(1)
+        m.fs.unit.inlet.flow_mass_comp[0, "giardia_lamblia"].fix(1)
+        m.fs.unit.inlet.flow_mass_comp[0, "eeq"].fix(1)
+        m.fs.unit.inlet.flow_mass_comp[0, "total_coliforms_fecal_ecoli"].fix(1)
+        m.fs.unit.inlet.flow_mass_comp[0, "viruses_enteric"].fix(1)
+        m.fs.unit.inlet.flow_mass_comp[0, "tss"].fix(1)
+
+        return m
+
+    @pytest.mark.unit
+    def test_toc_in_solute_list(self):
+        model = ConcreteModel()
+        model.db = Database()
+
+        model.fs = FlowsheetBlock(default={"dynamic": False})
+        model.fs.params = WaterParameterBlock(default={"solute_list": ["cryptosporidium", 
+                                                                        "giardia_lamblia",
+                                                                        "eeq"]})
+        with pytest.raises(ConfigurationError,
+                            match="TOC must be in solute list for Ozonation or Ozone/AOP"):
+
+
+            model.fs.unit = OzoneAOPZO(default={ "property_package": model.fs.params,
+                                                "database": model.db})
+
+
+    @pytest.mark.unit
+    def test_build(self, model):
+        assert model.fs.unit.config.database == model.db
```

Comment:
```suggestion
        assert model.fs.unit.config.database is model.db
```
Slight mod to this previously suggested by @bknueven that I started implementing. Instead of `==`, we're using `is`.